html/Axit/js/validation/contact-validation/contact-form-validation.js

```diff
@@ -0,0 +1,49 @@
+import {
+  checkName,
+  checkSubject,
+  checkMessage,
+  checkEmail,
+} from '../helper-functions.js';
+
+import {components} from './contact-components.js';
+
+export function addContactFormValidation() {
+  components.then((respose) => {
+    const {
+      contactForm,
+      errorMessages,
+      nameInput,
+      emailInput,
+      subjectInput,
+      messageTextArea,
+    } = respose;
+
+    contactForm.addEventListener('submit', () => {
+      event.preventDefault();
+      validateContactForm();
+    });
+
+    function validateContactForm() {
```

Comment:
It's better to declare static function outside `addContactFormValidation` to avoid function declaration on every emit